pkg/apis/marketplace/v1alpha1/operatorsource.go

```diff
@@ -0,0 +1,29 @@
+package v1alpha1
```

Comment:
You now have a file called `operatorsource.go` with only constants defining the phases.  Please move these constants into `operatorsource_types.go`.

Reply:
+1